src/pages/RFOX/components/Overview/Stats.tsx

```diff
@@ -1,112 +1,83 @@
 import { thorchainAssetId } from '@shapeshiftoss/caip'
 import { bn } from '@shapeshiftoss/chain-adapters'
 import { useMemo } from 'react'
 import { Text } from 'components/Text'
 import { fromBaseUnit } from 'lib/math'
 import { useAffiliateRevenueQuery } from 'pages/RFOX/hooks/useAffiliateRevenueQuery'
 import { useCurrentEpochMetadataQuery } from 'pages/RFOX/hooks/useCurrentEpochMetadataQuery'
-import { useTotalStakedQuery } from 'pages/RFOX/hooks/useGetTotalStaked'
+import { useRFOXContext } from 'pages/RFOX/hooks/useRfoxContext'
 import { selectAssetById, selectMarketDataByAssetIdUserCurrency } from 'state/slices/selectors'
 import { useAppSelector } from 'state/store'
 
+import { EmissionsPool } from './EmissionsPool'
 import { StatItem } from './StatItem'
+import { TotalStaked } from './TotalStaked'
 
 const gridColumns = { base: 1, md: 2 }
 
-type StatsProps = {
-  stakingAssetId: AssetId
-}
+export const Stats: React.FC = () => {
+  const { supportedStakingAssetIds } = useRFOXContext()
 
-export const Stats: React.FC<StatsProps> = ({ stakingAssetId }) => {
   const runeAsset = useAppSelector(state => selectAssetById(state, thorchainAssetId))
   const runeAssetMarketData = useAppSelector(state =>
     selectMarketDataByAssetIdUserCurrency(state, thorchainAssetId),
   )
 
-  const stakingAsset = useAppSelector(state => selectAssetById(state, stakingAssetId))
-  const stakingAssetMarketData = useAppSelector(state =>
-    selectMarketDataByAssetIdUserCurrency(state, stakingAssetId),
-  )
-
-  const currentEpochMetadataResult = useCurrentEpochMetadataQuery()
+  const currentEpochMetadataQuery = useCurrentEpochMetadataQuery()
 
-  const totalStakedUserCurrencyResult = useTotalStakedQuery<string>({
-    select: (totalStaked: bigint) => {
-      return bn(fromBaseUnit(totalStaked.toString(), stakingAsset?.precision ?? 0))
-        .times(stakingAssetMarketData.price)
-        .toFixed(2)
-    },
+  const affiliateRevenueQuery = useAffiliateRevenueQuery<string>({
+    startTimestamp: currentEpochMetadataQuery.data?.epochStartTimestamp,
+    endTimestamp: currentEpochMetadataQuery.data?.epochEndTimestamp,
   })
 
-  const affiliateRevenueResult = useAffiliateRevenueQuery<string>({
-    startTimestamp: currentEpochMetadataResult.data?.epochStartTimestamp,
-    endTimestamp: currentEpochMetadataResult.data?.epochEndTimestamp,
-    select: (totalRevenue: bigint) => {
-      return bn(fromBaseUnit(totalRevenue.toString(), runeAsset?.precision ?? 0))
-        .times(runeAssetMarketData.price)
-        .toFixed(2)
-    },
-  })
-
-  const emissions = useMemo(() => {
-    if (!affiliateRevenueResult.data) return
-    if (!currentEpochMetadataResult.data) return
+  const totalFeesCollectedUserCurrency = useMemo(() => {
+    if (!affiliateRevenueQuery.data) return
 
-    return bn(affiliateRevenueResult.data)
-      .times(currentEpochMetadataResult.data.distributionRate)
+    return bn(fromBaseUnit(affiliateRevenueQuery.data, runeAsset?.precision ?? 0))
```

Comment:
@NeOMakinG any reason why this was moved out of `select` ?

Reply:
I did that for naming purposes, felt weird calling it `affiliateRevenueResult` when it was user currency. Can add back and change the variable name from the query too I suppose. 

maybe like this?
```
  const totalFeesCollectedUserCurrencyQuery = useAffiliateRevenueQuery<string>({
    startTimestamp: currentEpochMetadataQuery.data?.epochStartTimestamp,
    endTimestamp: currentEpochMetadataQuery.data?.epochEndTimestamp,
    select: function getTotalFeesCollectedUserCurrency(totalRevenue) {
      return bn(fromBaseUnit(totalRevenue.toString(), runeAsset?.precision ?? 0))
        .times(runeAssetMarketData.price)
        .toFixed(2)
    },
  })
  ```

Reply:
Not a big deal, let's get this in instead of letting it getting stale 🙏

Reply:
That would make absolute sense @kaladinlight - going to merge for the time being for the sake of progression since extremely nitpick!